backend/routes/db.py

```diff
@@ -343,16 +275,164 @@ async def get_concepts_post_route(request: Request, id: Union[List[str], None] =
     return await get_concepts_route(request, id=id, table=table)
 
 
+sessions_cache: Dict = {}
+session_stale_hours = 24
```

Comment:
## Cache clearing: time-based vs LRU
I thought we were going to have a fixed amount of search results cached on the server? Are we not going to do that? I don't strongly object.